.ansible-lint

```diff
@@ -0,0 +1,3 @@
+exclude_paths:
+  - .github
+  - .ansible-lint
```

Comment:
i don't think it is good idea to exclude any of them

Reply:
It's just Ansible lint. Files in .github throw errors and are not important for ansible to work. So i excluded them.
.ansible-lint removed from exclude_paths